assets/src/edit-story/components/panels/preset/karma/colorPresets.karma.js

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../../karma/fixture';
+import { useInsertElement } from '../../../canvas';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../../../app/font/defaultFonts';
+
+describe('Panel: Color Presets', () => {
+  let fixture;
+
+  const ADD_BUTTON = 'Add color preset';
+  const EDIT_BUTTON = 'Edit color presets';
+  const EXIT_EDIT_MODE = 'Exit edit mode';
+
+  const addText = async () => {
+    const insertElement = await fixture.renderHook(() => useInsertElement());
+    await fixture.act(() =>
+      insertElement('text', {
+        font: TEXT_ELEMENT_DEFAULT_FONT,
+        content: 'Hello, presets!',
+      })
+    );
+  };
```

Comment:
Should we just use `fixture.editor.library.textAdd` here? So the text style actually matches what the users will experience?

Reply:
Yep, in this file makes sense, will change. In the text styles tests the `addText` is used for more things, will keep it there.